homeassistant/components/tuya/humidifier.py

```diff
@@ -138,6 +150,17 @@ def target_humidity(self) -> int | None:
 
         return round(self._set_humidity.scale_value(humidity))
 
+    @property
+    def current_humidity(self) -> int | None:
+        """Return the current humidity."""
+        if self._current_humidity is None:
+            return None
+
+        current_humidity: int | None = self.device.status.get(
+            self._current_humidity.dpcode
+        )
+        return current_humidity
```

Comment:
We need to scale the value, as every DP in Tuya can be scaled differently by the manufacturers, it also can return floats, while we only work with ints, so we need to round.

```suggestion
        if self._current_humidity is None:
            return None

        if (current_humidity := self.device.status.get(
            self._current_humidity.dpcode
        )) is None:
            return None

        return round(self._current_humidity.scale_value(current_humidity))
```